expyfun/io/_parse.py

```diff
@@ -73,3 +99,53 @@ def read_tab(fname, group_start='trial_id', group_end='trial_ok'):
             d[key] = [(these_vals[ii], these_times[ii]) for ii in idx]
         data.append(d)
     return data
+    
+    
+    def reconstruct_tracker(fname):
+        # read in raw data
+        raw = read_tab_raw(fname)
```

Comment:
My intent is for read_tab_raw to be the basic function that can either lead into read_tab (for people who want to split up trials) or to reconstructing tracker objects (where splitting up trials makes less sense.)